src/App.js

```diff
@@ -1,24 +1,19 @@
 import React from 'react';
 import logo from './logo.svg';
 import './App.css';
+import NavBar from './Components/NavBar.js'
+import UserProfile from './Components/UserProfile.js'
+import Posts from './Components/Post.js'
+import Contacts from './Components/Contacts.js'
+
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      {NavBar()}
```

Comment:
You don't need to invoke them. Think of them personalized divs. So you can use them like that. 

`<NavBar/>`
`<UserProfile/>`
etc